AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -685,11 +685,10 @@ - (void)startCrashProcessing {
    * counterintuitive, this is important because there are scenarios in some wrappers (i.e. RN) where
    * the application state is not ready by the time crash processing needs to happen.
    */
-  if (self.automaticProcessing &&
-      ([MSUtility applicationState] != MSApplicationStateActive &&
-       [MSUtility applicationState] != MSApplicationStateUnknown)) {
-        return;
-      }
+  if (self.automaticProcessing && [MSUtility applicationState] == MSApplicationStateBackground) {
```

Comment:
The comment indicates that there is no lifecycle for app extensions, so i guess if we remove the check for the "unknown" state, crash processing will not work for app extensions.

Reply:
I changed `!=` to `==`

Reply:
Yes disregard my comment